cli/sync_service/download.go

```diff
@@ -36,6 +36,15 @@ func ObjectDownLoad(org string, userPw string, objType string, objId string, fil
 		os.Setenv(config.HTTPRequestTimeoutOverride, "0")
 	}
 
+	// Call the MMS service over HTTP to get object metadata and determine the file size.
+	var objectMeta common.MetaData
+	metaUrlPath := path.Join("api/v1/objects/", org, objType, objId)
+	httpCode := cliutils.ExchangeGet("Model Management Service", cliutils.GetMMSUrl(), metaUrlPath, cliutils.OrgAndCreds(org, userPw), []int{200, 404}, &objectMeta)
```

Comment:
line 111- 116 also calls the api/v1/objects/org/objType/objID to get the metadata. Could you combine these somehow so it won't call syn service API to get metadata twice in ObjectDownLoad function?